### PR TITLE
[Relay][Frontend][Pytorch] Fixed ConvTranspose2D parsing

### PR DESCRIPTION
**Pull request by jwfromm**
There were some small bugs in the new pytorch frontend when parsing conv2d_transpose. This PR corrects them and adds related tests.

### CODE REVIEW

**Comment by jwfromm** [comment]
@masahi @alexwong can you take a look at this PR?

File: tests/python/frontend/pytorch/test_forward.py [hunk 1]
```diff
@@ -448,6 +448,33 @@ def forward(self, *args):
                  input_data=torch.randn((1, 8, 16, 16)))
 
 
+def test_forward_conv_transpose():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class ConvTranspose2D1(Module):
```
Comment (masahi) [inline tests/python/frontend/pytorch/test_forward.py]: New tests shouldn't introduce these wrapper modules. You ca use `torch.nn.ConvTranspose2d` directly. No need to have softmax. See

https://github.com/apache/incubator-tvm/blob/3aabbd9c30d247a31eb19ebf997d6074b14b5dd9/tests/python/frontend/pytorch/test_forward.py#L740-L742

Comment (jwfromm) [reply tests/python/frontend/pytorch/test_forward.py]: Ah good, the wrapperless version is much cleaner.

**Comment by masahi** [comment]
Thanks @jwfromm @tmoreau89
